images/nginx-drupal/drupal.conf

```
@@ -120,6 +120,14 @@ server {
     log_not_found off;
   }
 
+  ## Support for the securitytxt module
+  ## http://drupal.org/project/securitytxt.
+  ## RFC8615 standard path.
+  location = /.well-known/security.txt {
```

```suggestion
  location ~* ^/\.well-known/security\.txt(\.sig)?$ {
```

Actually, I think we need to add the path `/.well-known/security.txt.sig`

---

In my testing the `.txt.sig` file bootstrapped Drupal already so wasn't necessary, but being explicit here is nicer. 👍